CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * Ensure that organisation logos always print ([PR #4162](https://github.com/alphagov/govuk_publishing_components/pull/4162))
 * Fix some layout_header search visual bugs ([PR #4156](https://github.com/alphagov/govuk_publishing_components/pull/4156))
+* Upgrade to govuk-frontend v5.5.0 ([PR #4115](https://github.com/alphagov/govuk_publishing_components/pull/4160))
```

Comment:
Wondering if this is a breaking change? If it was deployed without the fix in `finder-frontend` it would cause something to look broken.